internal/exec/stack_processor_utils.go

```
@@ -2219,3 +2245,26 @@ func FindComponentsDerivedFromBaseComponents(
 
 	return res, nil
 }
+
+// validateImportPath performs security checks on import paths to prevent path traversal attacks and ensure paths are valid.
+// This is important when processing imports in stack manifests since they can reference external files that could potentially
+// be malicious if not properly validated. The function checks for empty paths, invalid filesystem characters, and directory
+// traversal attempts that could access files outside the intended directory structure.
+func validateImportPath(path string) error {
```

golangci-lint (1.62.2)

func `validateImportPath` is unused

(unused)